content-src/components/Search/Search.js

```diff
@@ -259,8 +277,10 @@ const Search = React.createClass({
           {formHistory.map(suggestion => {
             const active = (this.state.activeSuggestionIndex === suggestionsIdIndex);
             const activeEngine = this.getActiveEngine();
-            return (<li key={suggestion} role="presentation">
+            const suggestionIndex = suggestionsIdIndex++;
+            return (<li key={suggestion} role="option">
                   <a id={"history-search-suggestions-" + suggestionsIdIndex++ }
```

Comment:
do you need to remove the `++` here since you are doing it two lines above? I think you are incremementing it twice per row but it's like impossible to inspect in dev tools so I am not sure.